aries-site/src/examples/templates/forms/CustomizeExample.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Tile } from 'aries-core';
```

Comment:
Same question here for use of aries-core components. Would be an important discussion point.